src/microbe_stage/MicrobeAI.cs

```diff
@@ -243,7 +243,7 @@ private void ChooseActions(Random random, MicrobeAICommonData data, Microbe? sig
         }
 
         // If there are no threats, look for a chunk to eat
-        if (!microbe.CellTypeProperties.MembraneType.CellWall)
+        if (microbe.CanEngulf)
```

Comment:
You don't need to solve this now, but could you put in a TODO comment here that it'd be nice for the AI to know about colonies here and still consider engulfing things if it is in a colony that can engulf something?

Reply:
Added.